test/SlowTests/Issues/RavenDB-8451.cs

```diff
@@ -24,8 +24,29 @@ public RavenDB_8451(ITestOutputHelper output) : base(output)
 
         [Fact64Bit]
         public async Task CanRecoverEncryptedDatabase()
+        {
+            await CanRecoverEncryptedDatabaseInternal();
+        }
+
+        [Fact64Bit]
+        public async Task RecoveryOfEncryptedDatabaseWithoutMasterKeyShouldThrow()
+        {
+            try
+            {
+                await CanRecoverEncryptedDatabaseInternal(true);
+            }
+            catch (InvalidOperationException ioe)
+            {
+            }
```

Comment:
Can we use `Assert.Throws<>()` and check the exception message to be sure that it throws what we need